setup.py

```diff
@@ -10,11 +10,16 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "Django~=3.1.0",
+        "celery<4.0,>=3.1.15",
```

Comment:
Without `django-celery`, we should be able to use a modern version of Django and Celery.